std/conv.d

```diff
@@ -2341,13 +2341,15 @@ in
 body
 {
     import core.checkedint : mulu, addu;
+    import std.exception : enforce;
+
+    enforce!ConvException(!s.empty, "s must not be empty in integral parse");
```

Comment:
I meant the check is now before lines 2344 and 2345, whereas it used to be after.